bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/digitalinvoice/view/DigitalInvoiceOnboardingBarBottomAdapter.kt

```diff
@@ -0,0 +1,35 @@
+package net.gini.android.bank.sdk.capture.digitalinvoice.view
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import net.gini.android.bank.sdk.databinding.GbsDigitalInvoiceNavigationBarBottomBinding
+import net.gini.android.capture.view.InjectedViewAdapter
+
+interface DigitalInvoiceOnboardingBarBottomAdapter: InjectedViewAdapter {
+
+    fun setOnNextButtonClickListener(click: View.OnClickListener)
+}
+
+class DefaultDigitalInvoiceOnboardingBarBottomAdapter: DigitalInvoiceOnboardingBarBottomAdapter {
+
+    var viewBinding: GbsDigitalInvoiceNavigationBarBottomBinding? = null
+
+
+    override fun setOnNextButtonClickListener(click: View.OnClickListener) {
```

Comment:
[Request] The name `setGetStartedButtonClickListener()` reflects the purpose of the button better and is also consistent with the other onboarding navigation bar bottom adapter.